### PR TITLE
Add grid layout to search forms

### PR DESCRIPTION
**Pull request by kehh**
- Allow CSS classes to be associated with bundle placements on advanced search forms
- Make the `textareas` behave like `input[type='text']` in advanced search forms
- make text align left in all `.grid`s
- Remove source map from css
- add .editorconfig

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+indent_style = tab
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+
+# Matches the exact files either package.json or .travis.yml
```
Comment (leftclickben) [inline .editorconfig]: Incorrect.  Matches any .json file and any .gs file :-)

File: app/models/ca_search_forms.php [hunk 2]
```diff
@@ -1134,15 +1135,16 @@ public function getHTMLFormElements($po_request, $pa_form_data=null) {
 					$va_output[] = array(
 						'element' => $t_subject->htmlFormElementForSearch($po_request, $va_element['bundle_name'], array(
 							'values' => $pa_form_data,
-							'width' => (isset($va_element['settings']['width']) && ($va_element['settings']['width'] > 0)) ? $va_element['settings']['width'] : "100px",
-							'height' => (isset($va_element['settings']['height']) && ($va_element['settings']['height'] > 0)) ? $va_element['settings']['height'] : 1,
+							'width' => (isset($va_element['settings']['width']) && ($va_element['settings']['width'] > 0)) ? $va_element['settings']['width'] : 0,
```
Comment (leftclickben) [inline app/models/ca_search_forms.php]: Should the default really be `0`, or `null`?

**Comment by leftclickben** [comment]
Minor points only.

File: app/models/ca_search_forms.php (hunk 2 above)
Comment (kehh) [inline app/models/ca_search_forms.php]: accidentally committed change that I've now reverted.

File: .editorconfig (hunk 1 above)
Comment (kehh) [inline .editorconfig]: copypasta. Removed this part of the file as it doesn't even apply to this project.

**Pull request merged**